### ISSUE TITLE
Add `outline: 0` to flagged property/value combinations

### ISSUE THREAD

**Issue by Volker-E** [wikimedia/stylelint-config-wikimedia#89]
`outline: 0` is seen an [accessibility anti-pattern](https://webaim.org/blog/plague-of-outline-0/) and there are several places, where this was introduced for styling reasons and assumed lack of knowledge.
I'd propose for adding it to 'declaration-property-value-blacklist' which would need an extra override in case you're re-adding custom focus styles to your project.

**Comment by Volker-E** [comment]
@edg2s @Krinkle @jdlrobson your thoughts…?

**Comment by Volker-E** [comment]
Example I came from here: https://github.com/wikimedia/mediawiki-skins-MinervaNeue/blob/master/resources/skins.minerva.base.styles/ui.less#L148

**Comment by edg2s** [comment]
Seems reasonable. Are there any other examples in MW core, VE etc?

**Comment by Volker-E** [comment]
Oh yeah:
https://codesearch.wmflabs.org/search/?q=outline%3A%200&i=nope&files=&repos=

* lib/jquery.chosen
* ContentTranslation 
  * cardSearch
  * search tool
* DonationInterface
* Flow menu

and so forth and so on…

**Comment by edg2s** [comment]
As discussed, let's add a custom warning message for disabled properties that links to a wiki page (on wiki or github) outlining the reasoning for each disabled property.

**Comment by Volker-E** [comment]
@edg2s It would look like this
```
"declaration-property-value-blacklist": [ {
			"outline": ["/none/", "/0/"]
		},
			"message": "No `outline` is an accessibility anti-pattern. Only use when focussed state is ensured." ]
```
Not sure how to deal with two different blacklist items though…

**Comment by Volker-E** [comment]
@edg2s Let's move this forward, I'd suggest leaving a comment in the index.js instead. We've got a number of opinionated rules in here. And it's fair to expect certain engagement from devs with rules from a linter in our environment.
We've also been so far bringing the changes to the single repos, so that devs have an anchor point for orientation.
